common/Views/Banner.xaml

```diff
@@ -6,39 +6,41 @@
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     xmlns:commonviews="using:DevHome.Common.Views"
     mc:Ignorable="d">
-    <Grid Height="214">
+    <Grid x:Name="BannerGrid" Height="214">
         <Grid.Background>
             <ImageBrush Stretch="Fill" ImageSource="{x:Bind BackgroundSource, Mode=OneWay}" />
         </Grid.Background>
 
-        <!-- Set Min sizes so that image gets clipped as window narrows, rather than scaling down -->
-        <Image Stretch="Uniform" Source="{x:Bind OverlaySource, Mode=OneWay}" 
+        <!-- Background image -->
+        <Image Height="{Binding ElementName=BannerGrid, Path=Height}"
+               Stretch="Uniform" Source="{x:Bind OverlaySource, Mode=OneWay}"
                AutomationProperties.AccessibilityView="Raw"
-               Margin="0 0 0 0" MinHeight="214" MinWidth="{ThemeResource MaxPageContentWidth}"
+               Margin="0 0 0 0" MinWidth="{ThemeResource MaxPageContentWidth}"
                FlowDirection="{x:Bind FlowDirection}" />
 
+        <!-- Scrollable content -->
+        <ScrollViewer Height="{Binding ElementName=BannerGrid, Path=Height}">
+            <StackPanel VerticalAlignment="Center" Margin="38 0 0 0" Spacing="12">
+                <TextBlock MaxWidth="{x:Bind TextWidth}"
+                           HorizontalAlignment="Left"
+                           Style="{ThemeResource SubtitleTextBlockStyle}"
+                           Text="{x:Bind Title}"/>
+                <TextBlock MaxWidth="{x:Bind TextWidth}"
+                           HorizontalAlignment="Left"
+                           TextWrapping="Wrap"
+                           Text="{x:Bind Description}"/>
+                <Button Margin="0 15 0 0"

```

Comment:
I understand the margin.  Why the padding though?  Excessive padding tells me a stack panel isn't the correct container.

Reply:
The padding is relative to the Button element not the container. This determines how much space is between the button text and the button border. Let me know if you have other concerns.

![image](https://github.com/microsoft/devhome/assets/104940545/ce04e160-fe1e-464d-be04-4bd2ac8bd293)